### DISCUSSION TITLE
How do I clear a collection?

### DISCUSSION THREAD

**Discussion by ghost** [amoffat/supertag#31] [Q&A] [answered]
I've tried with
```
for i in "/mnt/myfiles"; do tag rmdir "$i"; done
Error: Not enough tags
```

**Comment by amoffat** [comment, marked as answer]
You can't clear the tags in a collection, you must remove the collection itself. You can access the underlying sqlite database though and do whatever you want though, see https://amoffat.github.io/supertag/tips_and_tricks.html?highlight=sqlite#browsing-the-database
